cmd/root.go

```diff
@@ -28,6 +28,7 @@ var (
 	flagConfiggin      string
 	flagLightOpinions  string
 	flagDarkOpinions   string
+	flagJSON           bool
```

Comment:
let's use an `--output` flag
It can have a default value of `human` and could also be `json` or `yaml` perhaps?